tools/clang_tidy/lib/clang_tidy.dart

```diff
@@ -229,16 +229,8 @@ class ClangTidy {
       if (job.result.exitCode == 0) {
         continue;
       }
-      if (job.exception != null) {
```

Comment:
Updated to remove this block.  It was only showing the first file linter errors because a linter failure makes the process fail, which populates the `job.exception`, and `break`s so doesn't show the other job failures.

Reply:
Nice catch